Login form/index.html

```diff
@@ -261,7 +261,6 @@ <h3>One of us ?</h3>
     }else{
       document.getElementById('useravatar').src = "";
       document.getElementById('username').innerHTML = "";
-      document.getElementById('username').innerHTML = "";
```

Comment:
@Abh1shekSingh since there is no major change in this PR that's why i can't give lvl2